Add envvar section to docs (#397)
OpenEnergyPlatform/open-MaStR: docs/advanced.rst

The data can then be written to an sql database. The type of the sql database is determined
by the parameter `engine` in the Mastr class (see :ref:`mastr module`). The possible databases are:

> * sqlite: This database will be stored in `$HOME/.open-MaStR/data/sqlite` (can be customized via the

Reviewer: ```suggestion
* sqlite: By default the database will be stored in `$HOME/.open-MaStR/data/sqlite/open-mastr.db` (can be customized via the
```

        engine = create_engine("postgresql://myusername:mypassword@localhost/mydatabase"
        db = Mastr(engine=engine)
..

Resulting data of download, post-processing and analysis is saved under `$HOME/.open-MaStR/data/<data-version>`.
Files that are suffixed with `_raw` contain joined data retrieved during :ref:`downloading <Download>`.
The structure of the data is described in :ref:`Data Description`.

Environment variables
^^^^^^^^^^^^^^^^^^^^^

There are some environment variables to customize open-MaStR:

.. list-table::
   :widths: 5 5 5
   :header-rows: 1

   * - variable
     - description
     - example
   * - SQLITE_DATABASE_PATH

>      - Path to the SQLite file. This allows to use to use multiple instances of the MaStR database.

Reviewer: ```suggestion
     - Path to the SQLite file. This allows to use to use multiple instances of the MaStR database. The database instances exist in parallel and are independent of each other.
```

>      - `/home/mastr-rabbit/.open-MaStR/data/sqlite/open-mastr-2022-12-15.db`

Reviewer: ```suggestion
     - `/home/mastr-rabbit/.open-MaStR/data/sqlite/your_custom_instance_name.db`
```
